app/views/users/two_factor_authentication_setup/index.html.erb

```diff
@@ -20,7 +20,7 @@
       <% @presenter.options.each do |option| %>
         <%= label_tag "two_factor_options_form_selection_#{option.type}",
                       class: "btn-border col-12 mb2 #{option.html_class}",
-                      hidden: option.html_class == 'hide' do %>
+                      hidden: option.html_class == 'display-none' do %>
```

Comment:
This should correspond to `WebauthnSelectionPresenter#html_class`, which it does.